documentation/snippet_embed_template.html

```diff
@@ -0,0 +1,27 @@
+<div id="container" style="min-width: 600px; min-height: 300px"></div>
+
+<!-- Monaco library script is loaded from: https://cdnjs.com/libraries/monaco-editor-->
+<script
+  src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.27.0/min/vs/loader.min.js"
+  integrity="sha512-SExj71Cw3B9C9EE8BC/ad3AKia5zQXDj/2SM4THgkeKh5GIFZhKM/R3uclUG8YZwJrjcVhydAlIHmfNvsBCKZA=="
+  crossorigin="anonymous"
+  referrerpolicy="no-referrer"
+></script>
+<script>
+  require.config({
```

Comment:
Does `loader.min.js` include requireJS or something? Wondering where this is coming from.

Reply:
Yeah I think so